code/modules/events/meteor_wave.dm

```diff
@@ -3,17 +3,20 @@
 /datum/round_event_control/meteor_wave
 	name = "Meteor Wave: Normal"
 	typepath = /datum/round_event/meteor_wave
-	weight = 4
+	weight = 7
 	min_players = 15
 	max_occurrences = 3
 	earliest_start = 25 MINUTES
 
 /datum/round_event/meteor_wave
 	startWhen = 6
-	endWhen = 66
+	endWhen = 66 // THE NUMBER OF DA BEAST
 	announceWhen = 1
 	var/list/wave_type
 	var/wave_name = "normal"
+	var/wave_amount = 3
+	var/waves_spawned = 0
+	var/wave_direction = NORTH
```

Comment:
Auto-doc these, they're inherited and modified and therefore are not private entries.